news/20200915.feature

```diff
@@ -0,0 +1 @@
+Add CMake variables for the selected printf and C library to the generated CMake module
```

Comment:
Add a newline to the end of this file please.

Reply:
Is [this](https://github.com/ARMmbed/mbed-tools/compare/b0e0c663fb9e74f4ce5d2e2d1c29e311d77c594f..2797d81a6358ac8c7bd4eb63918f31fdb670c305) ok?

Reply:
Looks good. Thanks.